lib/paypal_express/models/response.rb

```diff
@@ -78,19 +78,11 @@ def self.initial_payment_account_processor_id(kb_account_id, kb_tenant_id, token
       end
 
       def self.update_response_for_pending_payment(transaction_plugin_info)
```

Comment:
Maybe let's rename this to something more explicit (e.g. `cancel_pending_payment`)?